yt_whisper/utils.py

```diff
@@ -25,18 +25,37 @@ def format_timestamp(seconds: float):
 
     return (f"{hours}:" if hours > 0 else "") + f"{minutes:02d}:{seconds:02d}.{milliseconds:03d}"
 
+def break_line(line: str, length: int):
+    break_index = min(len(line)//2, length) # split evenly or at maximum length
 
-def write_vtt(transcript: Iterator[dict], file: TextIO):
+    # work backwards from that guess to split between words
+    # if break_index <= 1, we've hit the beginning of the string and can't split
+    while break_index > 1:
+        if line[break_index - 1] == " ":
+            break # break at space
+        else:
+            break_index -= 1
+    if break_index > 1:
+        # split the line, not including the space at break_index
+        return line[:break_index - 1] + "\n" + line[break_index:]
```

Comment:
This is the line that excludes the space between lines. i.e.
```
␣These agents have just begun learning,⏎
but they've already learned to chase and run away.⏎
```
The space (`␣`) at the beginning is generated by Whisper, but I think it might help evenness to either keep the space at the end of the line or remove the first one too.

Reply:
hmm, I'd rather remove it at the start too I think